src/components/BattleTV/BattleTV.module.css

```
@@ -1,192 +1,189 @@
+@import url('https://fonts.googleapis.com/css?family=Red+Hat+Display:400,900&display=swap');
+
 body,
 html {
   height: 100%;
   width: 100%;
-  background: #242a3c;
+  background: rgba(30, 30, 30, 0.87);
   color: #333;
-  overflow-x: auto;
 }
 
-.title {
-  margin-top: 3vh;
-  font-family: Montserrat;
-  font-weight: bold;
-  font-size: 3vw;
-  text-align: center;
-  left: 50%;
-  /* letter-spacing: 4.5px; */
+.header {
+  max-width: 1000px;
+  margin: 50px auto;
+  border-radius: 12px;
+  justify-content: center;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 10px 25px;
   color: #e8ecfd;
 }
 
-.battlecardHeader {
-  width: 80%;
-  margin: auto;
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  font-family: Montserrat;
-  font-style: normal;
-  font-weight: 600;
-  color: #ffffff;
-  margin: 20px auto 0 auto;
-  padding: 0 2%;
+.header__icon {
+  max-width: 50px;
+  height: auto;
+  opacity: 0.7;
 }
 
-.battlecardHeader .coinused,
-.battlecardHeader .destruction {
-  font-size: 0.8vw;
+.header__title {
+  font-size: 48px;
+  text-align: center;
+  text-transform: uppercase;
+  letter-spacing: 4.5px;
 }
 
-.battlecard {
-  display: flex;
-  flex-wrap: wrap;
-  /* min-width: 720px; */
-  margin: 1rem auto;
-  width: 80%;
-  border-radius: 0.5rem;
-  justify-content: space-between;
+.ranklist {
+  max-width: 1000px;
```

Do not harcode `px`.
```suggestion
  max-width: 50vw;
```
You can use `vw or vh` or `rem` or `em`.